### ISSUE TITLE
Running Scava-CrossMiner from Source code

### ISSUE THREAD

**Issue by razuiit** [crossminer/scava#451]
I am trying to run the scava from source code. I followed the instructions written (or demonstrated) on following pages: 
https://scava-docs.readthedocs.io/en/latest/others/deploy/Running-the-platform/
https://github.com/crossminer/crossminer/wiki/Running-the-platform
https://youtu.be/3Ry4KKfNdYg 

However, those instructions are either incomplete or hard to follow. For example, following the instructions in first link, i import the project in Neon3 RCP but project is imported with errors (as shown in Figure below). Additionally,  "org.ossmeter.platform.osgi/ossmeterfromfeature.product" is missing from crossminer/scava. 
In following the second link, I found the sbt version is incompatible with the OSSMeter project.
Is there any way i can follow to run Scava or CrossMiner from source code in eclipse?
![Errors](https://user-images.githubusercontent.com/20286837/71092100-80f04200-219e-11ea-9f66-509867992f96.png)

**Comment by davidediruscio** [comment]
Dear @razuiit thanks for your interest on the CROSSMINER project. It's evident we need to clean the documentation first by removing pages that are outdated. The steps that need to be followed to run the platform from the source code are detailed at:

https://scava-docs.readthedocs.io/en/latest/developers-guide/runing-from-sources/analysis-platform/#configure-the-analysis-platform

or 

https://github.com/crossminer/scava-docs/blob/master/docs/developers-guide/runing-from-sources/analysis-platform.md

Let us know how it goes.

**Comment by razuiit** [comment]
Hi Davide,

Partially successful this time. Steps are little opposite I guess, first need to set-up eclipse mars platform then need to import project.
Although, errors removed in most of the projects now, still its not running. I found few dependencies were missing when I validate the project. An image is attached to show those dependencies.

I am not sure if this is the problem due to the Rascal dependencies (my dominant guess). The versions of the dependencies described in docs (i.e. impulse_0.3.0.xxx.jar, rascal_eclipse_0.12.0.xxx.jar) don't actually exist on the website pointed. Hence, I tried multiple times with different versions, however, got the following error: 

[ERROR] Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'rascal-unstable' from location https://update.rascal-mpl.org/unstable/: Unable to read repository at https://update.rascal-mpl.org/unstable/content.xml. com.ibm.jsse2.util.h: PKIX path building failed: java.security.cert.CertPathBuilderException: PKIXCertPathBuilderImpl could not build a valid CertPath.; internal cause is:
[ERROR] java.security.cert.CertPathValidatorException: The certificate issued by CN=DST Root CA X3, O=Digital Signature Trust Co. is not trusted; internal cause is:
[ERROR] java.security.cert.CertPathValidatorException: Certificate chaining error

Help, please.

Abdul,
![error](https://user-images.githubusercontent.com/20286837/71375825-6c091980-25b7-11ea-89b4-849d6f1a0e7e.png)
